R/topic.R

```diff
@@ -0,0 +1,150 @@
+#' Gather topic from political discourses
+#'
+#' @param .data A data frame, "promises" data frame, or text vector.
```

Comment:
What is a "promises" data frame? Is this a new class? Or is it a data frame that is 'promised' e.g. https://stackoverflow.com/questions/41426358/what-is-a-promise-object-in-r. Something to do with callbacks and asynchronous programming perhaps? https://rstudio.github.io/promises/articles/promises_03_overview.html. Need to be clearer (again highlighting that a 'promise' is somewhat vague)

Reply:
Thank you for the feedback, I understand and have tried to clarify this on the documentation across. Please let me know if it is better or needs further clarification.